app/src/main/java/org/gdg/frisbee/android/event/EventOverviewFragment.java

```diff
@@ -220,39 +218,48 @@ private void updateGroupDetails(Chapter group) {
             return;
         }
 
-        Plus.PeopleApi.load(((GdgActivity) getActivity()).getGoogleApiClient(), group.getGplusId())
-            .setResultCallback(new ResultCallback<People.LoadPeopleResult>() {
-                @Override
-                public void onResult(People.LoadPeopleResult loadPeopleResult) {
-                    if (loadPeopleResult.getStatus().getStatusCode() == CommonStatusCodes.SUCCESS) {
-                        Person gplusChapter = loadPeopleResult.getPersonBuffer().get(0);
-                        if (gplusChapter.getImage().hasUrl()) {
-                            Picasso.with(getActivity()).load(gplusChapter.getImage().getUrl()).into(new Target() {
-                                @Override
-                                public void onBitmapLoaded(Bitmap bitmap, Picasso.LoadedFrom loadedFrom) {
-                                    if (!isAdded()) {
-                                        return;
-                                    }
-                                    BitmapDrawable logo = new BitmapDrawable(getResources(), bitmap);
-                                    mGroupLogo.setVisibility(View.VISIBLE);
-                                    mGroupLogo.setImageDrawable(logo);
-                                }
-
-                                @Override
-                                public void onBitmapFailed(Drawable drawable) {
-                                    mGroupLogo.setVisibility(View.INVISIBLE);
-                                }
-
-                                @Override
-                                public void onPrepareLoad(Drawable drawable) {
-
-                                }
-                            });
+        loadChapterImageInto(group.getGplusId(), mGroupLogo);
+
+        ((GdgActivity) getActivity()).setToolbarTitle(group.getShortName());
+    }
+
+    private void loadChapterImageInto(String gplusId, ImageView groupLogo) {
+        App.getInstance().getPlusApi().getImageInfo(gplusId).enqueue(new retrofit2.Callback<ImageInfo>() {
```

Comment:
@tasomaniac In https://github.com/gdg-x/frisbee/blob/583/anonymous_chapter_image/app/src/main/java/org/gdg/frisbee/android/api/PlusApiFactory.java#L18 there is the idlingResources configuration. Is that enough?